package.json

```diff
@@ -5,9 +5,9 @@
   "main": "app.js",
   "scripts": {
     "start": "nodemon app.js",
-    "test": "standard",
-    "lint": "standard",
-    "fix": "standard --fix"
+    "test": "standard && ejslint views/*.ejs && ejslint views/components/*.ejs",
+    "lint": "standard && ejslint views/*.ejs && ejslint views/components/*.ejs",
+    "fix": "standard --fix && ejslint views/*.ejs && ejslint views/components/*.ejs"
```

Comment:
Does the ejslint command automatically fix errors or does it just show errors? If it doesnt automatically fix errors then it shouldn't be in `npm run fix`.

Reply:
any errors that lint can't automatically fix are still outputted, so I thought it would be best to still output any ejs errors after `npm run fix`. I just thought it'd be misleading if there were still errors after the command and there was no alert.